slog-extlog-derive/src/lib.rs

```diff
@@ -560,35 +538,35 @@ fn impl_loggable(ast: &syn::DeriveInput) -> quote::Tokens {
     // Generate the actual log call based on the provided level.
     let match_gen = match level {
         Level::Critical => {
-            quote! { crit!(logger, #text; "log_id" => id_val, #(#fields, )* "details" => self) }
+            quote! { slog::crit!(logger, #text; "log_id" => id_val, #(#fields, )* "details" => self) }
```

Comment:
at the top, would it be better to put a:
```rust
use slog::{debug, info, warn, crit, error, trace}
```
and some other `slog` things used in this file?

Reply:
As we discussed - this `quote!` macro is building up the egenrated code for a `derive` impl - the `use` would need to be in the code that has `#[derive(ExtLoggable)]`.  It's much better to not require the user to `use` other macros in order to make your custom derive work.